libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/OfflineNavigator.kt

```diff
@@ -2,13 +2,19 @@ package com.mapbox.services.android.navigation.v5.navigation
 
 import com.mapbox.geojson.Point
 import com.mapbox.navigator.Navigator
+import com.mapbox.navigator.RouterParams
 import com.mapbox.navigator.TileEndpointConfiguration
 
 internal class OfflineNavigator @JvmOverloads constructor(
     private val navigator: Navigator,
     private val version: String = "",
     private val host: String = "",
-    private val accessToken: String = ""
+    private val accessToken: String = "",
+    private val navigatorVersion: String = "",
+    private val userAgent: String = "MapboxNavigationNative",
+    private val inMemoryTileCache: Int? = null,
+    private val mapMatchingSpatialCache: Int? = null,
+    private val threadsCount: Int = 2
```

Comment:
as I know `thread count` accepts null value, why we use 2 here?

Reply:
interestingly, this was for curl as well

Reply:
no, it was 2 threads, please check my comment